### ISSUE TITLE
ERROR (SyncWorker_2) [homeassistant.util.package] Unable to install package pyatv==0.8.2

### ISSUE THREAD

**Issue by terrycarlin** [postlund/pyatv#1253]
While trying to upgrade homeassistant, got this error message:
Unable to install package pyatv==0.8.2: ERROR: Cannot install aiohttp==3.7.4.post0 and pyatv because these package versions have conflicting dependencies

```log
ERROR (SyncWorker_2) [homeassistant.util.package] Unable to install package pyatv==0.8.2: ERROR: Cannot install aiohttp==3.7.4.post0 and pyatv because these package versions have conflicting dependencies
```
**To Reproduce**
pip3 upgrade homeassistant
**Expected behavior**
Homeassistant upgraded
**System Setup (please complete the following information):**
 - OS:  Raspbian Debian Version 10.7
 - Python: 3.8.5
 - pyatv: 0.8.2
 - Apple TV: Apple TV 4K
 - No Docker
 - Homeassistant: 2021.8.6
 - 
**Additional context**
removed and reinstalled homeassistant, pyatv, and aiohttp
No joy.

**Comment by postlund** [comment]
I will look into that, thanks for reporting!

**Comment by postlund** [comment]
This seems to happen because of some dependencies in audio-metadata (requiring too old versions):

```raw
ERROR: tbm-utils 2.6.0 has requirement attrs<19.4,>=18.2, but you'll have attrs 21.2.0 which is incompatible.
ERROR: audio-metadata 0.11.1 has requirement attrs<19.4,>=18.2, but you'll have attrs 21.2.0 which is incompatible.
```

As the dependencies comes from audio-metadata, it's hard to make any real fixes. One thing I can do however is to migrate to mutagen instead, something I have already considered in the past. mutagen has no other dependencies than standard python, so that would always work and also yield a lower dependency count.

**Comment by postlund** [comment]
For completeness sake, the dependency issue has been reported before:

https://github.com/thebigmunch/tbm-utils/issues/1

Would have been easier if that was fixed as mutagen seems a bit harder to use. Probably very flexible, but I'm using very little functionality and just need something that is easy to use.

**Comment by postlund** [comment]
This seems to be what I'm looking for:

https://github.com/beetbox/mediafile

**Comment by postlund** [comment]
Easy enough to switch to mediafile but seems to be an issue with wav files. Opened an issue about it: https://github.com/beetbox/mediafile/issues/49 Let's see if it can get a fix.

**Comment by postlund** [comment]
Fixed #1271, will be part of 0.9.0.
